pkg/util/k8sutil/k8sutil.go

```diff
@@ -365,11 +365,20 @@ func newEtcdPod(m *etcdutil.Member, initialCluster []string, clusterName, state,
 				Name:  "check-dns",
 				// In etcd 3.2, TLS listener will do a reverse-DNS lookup for pod IP -> hostname.
 				// If DNS entry is not warmed up, it will return empty result and peer connection will be rejected.
+				// In some cases the DNS is not created correctly so we time out after 10 minutes in this case.
```

Comment:
one last nit.
`In some cases the DNS is not created correctly so we time out after 10 minutes in this case.` ->
`In some cases the DNS is not created correctly so we need to time out after a given period`

Reply:
👍